src/app/components/ThemeProviderSCSSModules/themes/mundo/mundo.ts

```diff
@@ -0,0 +1,15 @@
+import '../../../fontFaces/reith-sans-bold.module.scss';
+import '../../../fontFaces/reith-sans-regular.module.scss';
+import '../../../fontFaces/reith-serif-medium.module.scss';
+import '../../../fontFaces/reith-serif-light.module.scss';
+import '../../../fontVariants/reith.module.scss';
+import './palette.module.scss';
+import '../../../fontScripts/latinWithDiacritics.module.scss';
```

Comment:
Quite important implementation detail here, these scss modules are imported and applied to the page, but because this is dynamically imported via loadable the theory is this css is correctly chunked with mundo only i.e. only css for the chosen theme is loaded